resources/views/components/avatar.blade.php

```diff
@@ -0,0 +1,8 @@
+<span class="{{ $avatarClasses }}">
+    {{ $content ?? $slot }}
+    @if(!$content && $slot->isEmpty())
```

Comment:
If the src image is null or the label is null, show the SVG

```suggestion
    @if (!$src || !$label)
```